src/main/java/org/gridsuite/study/server/service/ConsumerService.java

```diff
@@ -814,4 +812,30 @@ public Consumer<Message<String>> consumeStateEstimationStopped() {
     public Consumer<Message<String>> consumeStateEstimationFailed() {
         return message -> consumeCalculationFailed(message, STATE_ESTIMATION);
     }
+
+    public void consumeNetworkExportFinished(Message<String> msg) {
+        Optional.ofNullable(msg.getHeaders().get(NETWORK_UUID, String.class))
+                .map(UUID::fromString)
+                .ifPresent(networkUuid -> {
```

Comment:
Why working with the networkUuid here ? 
It should be all about the exportUuid I think. The networkUuid shouldn't even be present in the notification. Should also be fixed in network-modification